relationship_server.py

```diff
@@ -13,38 +13,45 @@
 logger = logging.getLogger(__name__)
 
 
-'''
-    Utliity method to convert keys and values in a dictionary to lowercase.
-'''
-def makeDictLowercase(d):
-    lowerCaseDict = dict()
+def make_dict_lowercase(d):
+    """
+        Utliity method to convert keys and values in a dictionary to lowercase.
+    """
+    lower_case_dict = dict()
     for k in d.keys():
-        lowerCaseDict[k.lower()] = d[k].lower()
-    return lowerCaseDict
-
-'''
-    Utility to convert string to Title Case
-    Input: this is a sentence.
-    Output: This Is A Sentence.
-'''
-def toTitleCase(string):
+        lower_case_dict[k.lower()] = d[k].lower()
+    return lower_case_dict
+
+
+def to_title_case(string):
+    """
+        Utility to convert string to Title Case
+        Input: this is a sentence.
+        Output: This Is A Sentence.
+    """
     return " ".join(w.capitalize() for w in string.split(" "))
 
-'''
-    Loading JSON that has alias / acronym : country name mapping.
-'''
-def loadCountryAcronymJson():
-    with urllib.request.urlopen("https://raw.githubusercontent.com/rohanrmallya/coronaIndia/master/data/countries_acronym_aliases_flattened.json") as url:
+
+def load_country_acryonym_json(
+    download_url: str = "https://raw.githubusercontent.com/rohanrmallya/coronaIndia/master/data/countries_acronym_aliases_flattened.json",
+) -> None:
+    """
```

Comment:
Consider writing [Google style docstrings](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html)? These are machine readable and lot of devs e.g. me are very comfortable reading them.

Reply:
Have tried my best to do so. Let me know if I can improve it further. :+1:

Reply:
The indentations etc. are a little noisy. But looks sort of okay otherwise for now. I'd recommend reading through some simple, more mature codebases on how to do this right.